packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```
@@ -177,18 +179,22 @@ EOF
 %{foreman_proxy_bundlerd_dir}/%{plugin_name}.rb
 %exclude %{gem_cache}
 %{gem_spec}
-%{foreman_proxy_bundlerd_dir}/smart_proxy_salt_core.rb
-%{smart_proxy_dynflow_bundlerd_dir}/smart_proxy_salt_core.rb
 %config(noreplace) %{salt_config_dir}/foreman.yaml
 %config %{_root_sysconfdir}/cron.d/%{gem_name}
 %exclude %{gem_instdir}/etc
 %exclude %{gem_instdir}/cron
+%attr(-,foreman-proxy,foreman-proxy) %{salt_state_grains_dir}/autosign_key
```

```suggestion
%{salt_proxy_runners_dir}
%{salt_proxy_reactors_dir}
%attr(-,foreman-proxy,foreman-proxy) %{salt_state_grains_dir}/autosign_key
```

otherwise the runners and reactors in `/usr/share` are not picked up.